ParmDB/ParmSet.h

```diff
@@ -56,11 +56,11 @@ namespace BBS {
     ParmSet();
 
     // Add a parm for the given ParmDB. If not existing in the parmDB,
-    // it will be added to the ParmDB when its values are written.
+    // it will be added to the ParmDB when its values are writtn.
```

Comment:
Curious why you think this is an improvement ;)

Reply:
Haha I did not do this knowingly :). I'll revert it. Good catch :).